backend/openapi-specs/graph-service-webhook.yaml

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
why do we have a yaml instead of a json doc?

Reply:
Mainly because I wanted to put comments in the file and you can't do that with JSON